WordPressAuthenticator/Signin/LoginEmailViewController.swift

```diff
@@ -522,6 +532,9 @@ extension LoginEmailViewController {
         googleExistingUserNeedsConnection(email)
     }
 
+    /// After a successful Google Sign In, this method gets called when the user
+    /// has enabled 2-factor authentication for their WordPress.com account.
+    ///
```

Comment:
This is documentation I forgot to update for the previous segue removal, `.show2fa`